benchmark/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
```

Comment:
this file should not be needed

Reply:
It's for convenient for local testing. If more benchmarks are added that require services this would make for a simple setup, although port management might be a bit annoying.